releasenotes/notes/make-missing-tags-errors-c4cd44a12d22e7bc.yaml

```
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+deprecations:
+  - |
+    SNMP profiles containing metric_tags without a `tag` field
+    specified will now show an error and be ignored, as with any
+    other profile syntax error.
```

Non-blocking suggestion:

```suggestion
    SNMP profiles containing metric_tags without a specified `tag` field
    will now show an error and be ignored, similar to other profile syntax errors.
```